modules/installation/pages/install-proxy.adoc

```diff
@@ -1,8 +1,125 @@
 [[installation-proxy]]
-= {productname} {productnumber} Proxy
+= Install {productname} Proxy
 
+This chapter provides the required KVM settings for installation of {sles} media as the base for {productname} Proxy.
+A kernel virtual machine KVM combined with Virtual Machine Manager ([command]``virt-manager``) will be used as a sandbox for this installation.
 
 
 
+[[installation-proxy-requirements]]
+== SLES KVM Requirements
 
-Watch this space ...
+Enter the following settings when creating a new virtual machine using [command]``virt-manager`` (replace [literal]``<version>`` with the actual version string):
+
+[cols="1,1", options="header"]
+|===
+| KVM Settings for SLES
+| Installation Method: | Local install media (ISO image or CDROM)
+| OS:                  | Linux
+| Version:             |``SLE-``[literal]``<version>````-Server-x86_64-GM-DVD1.iso``
+| Memory:              | _Test Server_ Minumum 2{nbsp}GB
+|                      | _Production Server_ Minimum 8{nbsp}GB
+| CPUs:                | 2
+| Storage Format:      | ISO 3{nbsp}GB
+| Disk Space:          | 230{nbsp}GB split between
+|                      | [path]``/`` _(root)_ Minimum 24{nbsp}GB
+|                      | (Virtual Disk 1) [path]``/srv`` Minimum 100{nbsp}GB
+|                      | (Virtual Disk 2) [path]``/var/cache`` _(Squid)_ Minimum 100{nbsp}GB
+| Name:                | example-proxy
+| Network              | Bridge br0
+|===
+
+
+
+[[installation-proxy-sles-settings]]
+=== SLES KVM Settings
+
+This section provides guidance on installation of {productname} Proxy
+utilizing the full installation media with KVM and [command]``virt-manager``.
```

Comment:
Extra line break here.

Reply:
I would probably reword as "This sections guides you through" or similar, to get rid of the zombie "provides guidance".

Reply:
submitting externally